Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureRequest.h

```diff
@@ -206,6 +237,24 @@ typedef NS_ENUM(NSInteger, BTThreeDSecureRequestedExemptionType) {
  */
 @property (nonatomic, nullable, weak) id<BTThreeDSecureRequestDelegate> threeDSecureRequestDelegate;
 
+/**
+ Optional. The interface types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge.
+
+ Defaults to `BTThreeDSecureUITypeBoth`
+ */
+@property (nonatomic, assign) BTThreeDSecureUIType uiType;
+
+/**
+ Optional.  List of all the render types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge.
+
+ Defaults to `BTThreeDSecureRenderTypeOTP`, `BTThreeDSecureRenderTypeHTML`, `BTThreeDSecureRenderTypeOOB`,
+ `BTThreeDSecureRenderTypeSingleSelect`,`BTThreeDSecureRenderTypeMultiSelect`
+
+ - Note: When using `BTThreeDSecureUITypeBoth` or `BTThreeDSecureUITypeHTML`, all `renderType` options must be set.
+ When using `BTThreeDSecureUITypeNative`, all `renderType` options except `BTThreeDSecureRenderTypeHTML` must be set.
+ */
+@property (nonatomic, copy) NSArray<BTThreeDSecureRenderType> *renderType;
```

Comment:
To match Android naming

```suggestion
@property (nonatomic, copy) NSArray<BTThreeDSecureRenderType> *renderTypes;
```

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1039/commits/27c9c2fff67086b663e7a56ba6327edb98d959c8